index.html

```diff
@@ -18,18 +18,39 @@
                 <div class="logo"><img src="./images/logo.png" alt="Logo"></div>
                 <nav class="myNav">
                     <ul class="navs">
-                        <li class="navList" id="hello"><a href="#hello">Hello</a></li>
+                        <li class="navList"><a href="#hello">Hello</a></li>
                         <li class="navList"><a href="#portfolio">Portfolio</a></li>
                         <li class="navList"><a href="#about">About</a></li>
-                        <li class="navList"><a href="#contact">Contact</a></li>
+                        <li class="navList"><a href="#contact" id="id_contact">Contact</a></li>
                     </ul>
```

Comment:
- [ ] We shall together check out the designs for the two screenshots below: That of Figma and yours.


| Yours | Figma |
| --- | --- |
| ![Screenshot from 2022-11-15 16-35-46](https://user-images.githubusercontent.com/63087491/201961333-ebe6ccdc-de13-4858-8429-4c9fd1ea61a3.png) | ![Screenshot from 2022-11-15 16-35-16](https://user-images.githubusercontent.com/63087491/201961375-9d63f93d-8fb1-48da-b8da-2747479b25be.png) |

- It seems the backgrounds are completely different. We shall once more make sure that our design matches that of Figma(font, colors, images, text, margins)

Reply:
_Thanks so much.
to be sincere with you I knew this was going to be returned 
I just had to submit it for better understanding. I don't know how to add this image to the mobile menu
I have tried several ways, but have not gotten it.
the images are separate files they are not all together on the Figma template.
If there is any understanding you can share with me, I will appreciate it.
thanks 🙏  😄_